wire-ios/WireCommonComponents/UIColor+AccentColor.swift

```diff
@@ -34,12 +34,16 @@ public enum AccentColor: Int16, CaseIterable {
         return AccentColor.allSelectable().randomElement()!
     }
 
+    public init() {
+        self = .allCases[0]
+    }
```

Comment:
To read this from some place, I find this more implicit and one needs to look up what `= AccentColor()` case is chosen compared to `= .blue`.

Reply:
Also here I wanted to offer a default initialiser. I'll replace it by a static `default` property.